internal/operands/template-validator/reconcile_test.go

```diff
@@ -83,6 +88,207 @@ var _ = Describe("Template validator operand", func() {
 		ExpectResourceExists(newValidatingWebhook(namespace), request)
 	})
 
+	Context("with different requests", func() {
+
+		nodeAffinity := &core.NodeAffinity{
```

Comment:
Can you assign to these variables in `BeforeEach()`, so the test are independent?

Reply:
Right, I didn't notice that. You can leave it as is.